pkg/app/appdisc/const.go

```diff
@@ -0,0 +1,7 @@
+package appdisc
+
+// ChangeValue keys.
+const (
+	ConnCountValue     = "conn_count"
```

Comment:
Could you please add comments to these? Not clear what are these consts for

Reply:
Will do, thank you.